src/discord-cluster-manager/leaderboard_db.py

```diff
@@ -198,17 +219,20 @@ def get_leaderboard(self, leaderboard_name: str) -> int | None:
             return None
 
     # TODO: add GPU type
-    def get_leaderboard_submissions(self, leaderboard_name: str) -> list[SubmissionItem]:
+    def get_leaderboard_submissions(
+        self, leaderboard_name: str, gpu_name: str
```

Comment:
Can we maybe make this optional? Or create a new method that filters based on key: value pairs, etc. I would like this as pure "get all"

Reply:
But get all is weird -- if you want to get all you can just select all GPUs, otw I don't like this being a default option. Because the selection is in the UI now, I like the the user having to select what GPUs they want.